packages/app-store/slack/DESCRIPTION.md

```diff
@@ -0,0 +1,7 @@
+---
+items:
+  - slack1.png
```

Comment:
Can you add a couple more screenshots? We can have one of the installation page where Slack credentials are added.

Reply:
@rkreddy99